test/test_open.py

```diff
@@ -129,8 +147,17 @@ def test_file_perms(self, safer_open):
             fp.write('hello')
         assert FILENAME.read_text() == 'hello'
         mode = os.stat(FILENAME).st_mode
-        assert mode in (0o100664, 0o100644), stat.filemode(mode)
-        new_mode = mode & 0o100770
+
+        # UNIX systems view and manipulate file permissions as bits
```

Comment:
Kill the comment, "everyone" knows this.

In general, you should be writing for a competent practitioner, but don't assume brilliance.